advisor/src/funTest/kotlin/OsvFunTest.kt

```diff
@@ -33,6 +32,7 @@ import org.ossreviewtoolkit.model.Identifier
 import org.ossreviewtoolkit.model.Package
 import org.ossreviewtoolkit.model.config.AdvisorConfiguration
 import org.ossreviewtoolkit.model.readValue
+import org.ossreviewtoolkit.utils.test.getAssetFile
```

Comment:
Sorry, I missed the function names in this commit message title, which would be put into backticks as well. Will approve anyway.